### ISSUE TITLE
No module named 'lie_learn.representations.SO3.irrep_bases

### ISSUE THREAD

**Issue by skeydan** [QUVA-Lab/escnn#24]
Running `from escnn.group import *`, I get the following error:

```
>>> from  escnn.group import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/key/.virtualenvs/pt1.13.1/lib64/python3.11/site-packages/escnn-1.0.3-py3.11.egg/escnn/group/__init__.py", line 22, in <module>
    from .groups.factory import *
  File "/home/key/.virtualenvs/pt1.13.1/lib64/python3.11/site-packages/escnn-1.0.3-py3.11.egg/escnn/group/groups/__init__.py", line 2, in <module>
    from .factory import *
  File "/home/key/.virtualenvs/pt1.13.1/lib64/python3.11/site-packages/escnn-1.0.3-py3.11.egg/escnn/group/groups/factory.py", line 8, in <module>
    from .so3group import SO3
  File "/home/key/.virtualenvs/pt1.13.1/lib64/python3.11/site-packages/escnn-1.0.3-py3.11.egg/escnn/group/groups/so3group.py", line 8, in <module>
    from .so3_utils import *
  File "/home/key/.virtualenvs/pt1.13.1/lib64/python3.11/site-packages/escnn-1.0.3-py3.11.egg/escnn/group/groups/so3_utils.py", line 1, in <module>
    from lie_learn.representations.SO3.wigner_d import wigner_D_matrix
  File "/home/key/.virtualenvs/pt1.13.1/lib64/python3.11/site-packages/lie_learn-0.0.1.post1-py3.11.egg/lie_learn/representations/SO3/wigner_d.py", line 5, in <module>
    from lie_learn.representations.SO3.irrep_bases import change_of_basis_matrix
ModuleNotFoundError: No module named 'lie_learn.representations.SO3.irrep_bases'

```

I've tried various ways of installing `escnn` as well as `lie_learn`, with no success. (The last method I tried was cloning the repo and doing `python setup.py install`.)
My Python version is 3.11.1.

I'm aware of a similar issue in lie-learn, but it is 2.5 years old, and unsolved: https://github.com/AMLab-Amsterdam/lie_learn/issues/16

Could someone please take a look? Thanks!

**Comment by Gabri95** [comment]
Hi @skeydan,

Thanks for reporting this!

I have recently found out that `lie_learn` is not compatible with python 3.10 or higher indeed.
I am already working on removing the dependency on `lie_learn`, but for the moment this is indeed a problem.

In the meantime, please use python 3.8

I hope to upload a fix soon

Best,
Gabriele

**Comment by kartikchincholikar** [comment]
Google Colab recently (10th March 2023) updated the default python version to 3.9. This made me run into this error. 
Running the following line at the start of the notebook helped fix the problem:
`!pip install git+https://github.com/AMLab-Amsterdam/lie_learn escnn`

**Comment by ashiq24** [comment]
@Gabri95  I am uncertain about the current status of the issue at hand, and whether all dependencies associated with `lie_learn` have been successfully removed. However, I recently faced a similar challenge that was addressed by modifying the `setup.py` file and installing a specific version of `lie_learn`. 

It would be of immense value to the community to have a more comprehensive resolution to this problem.
